Gemfile

```diff
@@ -4,14 +4,14 @@ gem 'rake'
 gem 'compass'
 gem 'jekyll'
 gem 'octopress-hooks'
-gem 'rdiscount'
-gem 'pygments.rb'
-gem 'RedCloth'
+gem 'kramdown-parser-gfm'
```

Comment:
Check out https://github.blog/2016-04-01-a-look-behind-our-decision-to-standardize-on-a-single-markdown-engine-for-github-pages/